compiler/natives/src/runtime/runtime_test.go

```diff
@@ -0,0 +1,80 @@
+//go:build js
+// +build js
+
+package runtime
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/gopherjs/gopherjs/js"
+)
+
+func Test_parseCallstack(t *testing.T) {
+	tests := []struct {
```

Comment:
It would be useful to add another couple of test cases:
  - a stack trace captured with sourcemaps enabled
  - a stack trace generated by nodejs.

Reply:
Agreed. I can easily add Mozilla, too, I think, while I'm at it.